### PR TITLE
framework/task_manger/broadcast: change a cb function name and argument

### PR DESCRIPTION
**Pull request by jeongchanKim**
broadcast also uses callback prototype as "void func(void *)"

### CODE REVIEW

File: framework/src/task_manager/task_manager_core.c [hunk 1]
```diff
@@ -651,11 +647,11 @@ int task_manager(int argc, char *argv[])
 			break;
 
 		case TASKMGT_SET_BROADCAST_HANDLER:
```
Comment (tyha1115) [inline framework/src/task_manager/task_manager_core.c]: TASKMGT_SET_BROADCAST_HANDLER will be changed into TASKMGT_SET_BROADCAST_CB

**Comment by seinfra** [comment]
Target : [952dd70bed99f0e37b4bd0096267bf867f3e775a] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [952dd70bed99f0e37b4bd0096267bf867f3e775a] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [a424844abae9f729705f3bc0af2f5e839c107c38] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [a424844abae9f729705f3bc0af2f5e839c107c38] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [a12785b873b4ef65b4759ec3a73f8f21aeb98be0] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [a12785b873b4ef65b4759ec3a73f8f21aeb98be0] - Code Rule Check OK.

**Pull request merged**